src/pca_report_library/templates/generate_template.py

```diff
@@ -79,11 +77,8 @@ def main():
         print("Copying Asset Folder for LaTeX report to current dir....")
 
     for i in os.listdir(temp_working_dir):
```

Comment:
The variable name `i` is incredibly unhelpful.

```suggestion
    for temp_entry in os.listdir(temp_working_dir):
```